modules/ROOT/edits/prepare-to-install-service-mesh.adoc

```diff
@@ -0,0 +1,133 @@
+= Reviewing Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, installation prerequisites
+
+image::prepare-to-install-breadcrumb.png[]
+
+Before you download and install Anypoint Service Mesh, ensure that you review and fulfill the software, licensing, and permissions and roles requirements described in this section. 
+
+== Audience Requirements
+
+Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you must have a working knowledge of Istio and Kubernetes to be able to install and use Anypoint Service Mesh.
+
+
+== Software Requirements
+
+Your Anypoint Service Mesh installation requires the following applications and Kubernetes environments:
+
+* Kubernetes 1.12+ or a newer cluster, with RBAC (Role-Based Access Control) enabled
+* Helm 3.0.0 or later
++
+*Tip:* To verify the installed version, use the `helm version` command. 
++
+For more information about how to install Helm, see the Helm installation documentation.
+* Kubectl version 1.12 or later, compatible with the cluster, installed on your local machine
+*Tip:* To verify installed version, use the `kubectl version` command.
+* One of the following environments on which to run Kubernetes:
+** Google Kubernetes Engine (GKE)
+** Amazon EKS
+** Azure Kubernetes Service (AKS)
+* Not installed Istio in the cluster
+* If you want to enable automatic sidecar injection, you must additionally:
+. Install Kubernetes 1.12 with the admissionregistration API.
+. Verify the installation using the following the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
+. Configure the kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”.
+. Add and list ValidatingAdmissionWebhook admission controllers in the correct order.
+* Verify that the Kubernetes api-server has network connectivity to the webhook pod and that the Kubernetes cluster follows other xref:https://istio.io/docs/ops/configuration/mesh/webhook/#verify-dynamic-admission-webhook-prerequisites[Istio network prerequisites.] 
```

Comment:
Istio and sidecar injection is not related. They are separate points. Can't join the together. And since there is only one point where the users have to check what is not installed, I'd leave it as is.